source/main.cpp

```diff
@@ -413,6 +425,12 @@ std::istream& GetLine(std::istream& is, std::string& t)
     }
 }
 
+u64 hex_to_u64( std::string value) {
```

Comment:
oops not yet, it's used for converting the titleId (string) to a u64 for passing, mainly to getProductCode to see if the title is installed or not.